envisage/ids.py

```diff
@@ -0,0 +1,56 @@
+# (C) Copyright 2007-2019 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+
+""" This module redefines the Extension Point IDs and Service IDs defined on
+Plugins provided by Envisage. Note that this module does not contain IDs
+defined by all of the Plugins available in Envisage.
+
+The Plugins themselves remain the ground truth for the IDs. This module is
+simply a convenient location from which the user can import the IDs.
+"""
+
+#### Extension Points #########################################################
+
+#: Ext. Pt. to contribute preference files, defined on the ``CorePlugin``.
```

Comment:
Any reason not to spell this out as "Extension Point"? I think that would look better in the docs - there's no real advantage to abbreviating.